From decb355247c4ba4b876997f55c27ec3f55dbacd2 Mon Sep 17 00:00:00 2001
From: Mohamed Omar Asaker <mohamed.omarasaker@arm.com>
Date: Mon, 23 Jan 2023 13:25:28 +0000
Subject: [PATCH] Platform: corstone1000: Increase number of assets

As Corstone1000 stores at boot time few efi variables.
Therefore, number of assets is increased to compansate this early usage.

Note: Adding platform customized configs to config_tfm.h
      More information see:
https://tf-m-user-guide.trustedfirmware.org/configuration/header_file_system.html

Signed-off-by: Mohamed Omar Asaker <mohamed.omarasaker@arm.com>
Upstream-Status: Pending [Not submitted yet]
---
 platform/ext/target/arm/corstone1000/config_tfm_target.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/platform/ext/target/arm/corstone1000/config_tfm_target.h b/platform/ext/target/arm/corstone1000/config_tfm_target.h
index bf8d2f95f7..e968366639 100644
--- a/platform/ext/target/arm/corstone1000/config_tfm_target.h
+++ b/platform/ext/target/arm/corstone1000/config_tfm_target.h
@@ -16,4 +16,12 @@
 #undef PLATFORM_SERVICE_OUTPUT_BUFFER_SIZE
 #define PLATFORM_SERVICE_OUTPUT_BUFFER_SIZE    256
 
+/* The maximum number of assets to be stored in the Internal Trusted Storage. */
+#undef ITS_NUM_ASSETS
+#define ITS_NUM_ASSETS       20
+
+/* The maximum number of assets to be stored in the Protected Storage area. */
+#undef PS_NUM_ASSETS
+#define PS_NUM_ASSETS        20
+
 #endif /* __CONFIG_TFM_TARGET_H__ */
-- 
2.25.1

